code/modules/clothing/under/accessories.dm

```diff
@@ -110,6 +111,11 @@
 					else
 						user.visible_message("[user] pins \the [src] on [M]'s chest.", \
 											 "<span class='notice'>You pin \the [src] on [M]'s chest.</span>")
+						if(!commended)
+							var/input = stripped_input(user,"Please input a reason for this commendation, it will be recorded by Nanotrasen.", ,"", 50)
+							if(input)
```

Comment:
that will mean that the commendation box will pop up even if the medal's been used before though